packages/amplify-provider-awscloudformation/src/root-stack-builder/types.ts

```diff
@@ -0,0 +1,17 @@
+import * as cdk from '@aws-cdk/core';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as iam from '@aws-cdk/aws-iam';
+
+export interface AmplifyRootStackTemplateProps {
+  Parameters?: { [k: string]: cdk.CfnParameter };
+  Resources?: AmplifyRootStackResourceProps;
+  Outputs?: { [k: string]: cdk.CfnOutput };
+  Mappinngs?: { [k: string]: cdk.CfnMapping };
```

Comment:
`Mappings` typo, these should be `Record` types but not really needed based on previous comments